include/gbinder_reader.h

```diff
@@ -169,6 +169,11 @@ gboolean
 gbinder_reader_skip_string16(
     GBinderReader* reader);
 
+const void*
+gbinder_reader_read_byte_array(
+    GBinderReader *reader,
+    gint32 *len); /* since 1.0.12 */
```

Comment:
I prefer `gsize*` for the length returned by the reader (even though it's always serialized as int32). If nothing else, that would be consistent with `gbinder_reader_read_hidl_vec`.